### PR TITLE
DDF-1407 Fix view destroyed errors in workspaces

### PR DESCRIPTION
**Pull request by harrison-tarr**
The code originally had 2 listeners listening to the same event
and caused a parent to rerender its children while the children
tried to rerender themselves. Now the rendering is handled by
the parent.
Also cleaned up some backbone to make it more standard.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/152)

### CODE REVIEW

**Comment by harrison-tarr** [comment]
@brendan-hofmann @pklinef @stustison @andrewkfiedler

**Comment by brendan-hofmann** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

<sup>**[catalog/ui/search-ui/standard/README.md, line 27 \[r1\]](https://reviewable.io:443/reviews/codice/ddf/152#-JyE4kr8VsQDpjyJU_Vs)** ([raw file](https://github.com/codice/ddf/blob/904b70fc98df8d46023a2010c2da0482c51e5b44/catalog/ui/search-ui/standard/README.md#L27)):</sup>
This isn't strictly accurate... The real issue is that you need the trailing slash when going to something like `/search/standard/` <-- that one. This is related to how the grunt proxy determines url roots; the trailing slash makes it treat the whole thing as the root, otherwise it uses `localhost:8282` as the root when proxying, which results in incorrect urls. There are ways to work around this behavior in `server.js` and `server-impl.js` as well, but I'm not convinced the current behavior is actually incorrect.

I suggest rewording it to something like the below instead:
```
Open http://localhost:8282/ to test and debug. 
Note that URLs (such as http://localhost:8282/search/standard/) must be terminated with the trailing slash in order for the grunt proxy to determine the url root correctly.
```

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/152)

**Comment by andrew-fiedler** [comment]
:+1:

**Comment by brendan-hofmann** [comment]
Successfully built, tested, and verified. :+1:

**Comment by pklinef** [comment]
:+1:
